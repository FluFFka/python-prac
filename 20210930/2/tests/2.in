789456789, 789456999
