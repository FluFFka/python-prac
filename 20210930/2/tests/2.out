[789456793, 789456799, 789456803, 789456809, 789456869, 789456893, 789456911, 789456919, 789456923, 789456949, 789456953, 789456977]
